gp-templates/translations.php

```diff
@@ -54,6 +54,30 @@
 	<select name="bulk[priority]" id="bulk-priority" class="hidden">
 	<?php
 	foreach( GP::$original->get_static( 'priorities' ) as $value => $label ) {
+		$label_hidden = [
+			'hidden' => _x( 'hidden', 'Priority', 'glotpress' ),
+		];
+		$label_low = [
+			'low' => _x( 'low', 'Priority', 'glotpress' ),
+		];
+		$label_normal = [
+			'normal' => _x( 'normal', 'Priority', 'glotpress' ),
+		];
+		$label_high = [
+			'high' => _x( 'high', 'Priority', 'glotpress' ),
+		];
```

Comment:
Same as above, just one array and before the loop.

Reply:
Done